pants_plugins/sendwave/pants_node/package.py

```diff
@@ -125,7 +125,7 @@ async def get_node_package_digest(field_set: NodeProjectFieldSet) -> Digest:
         ),
     )
     logger.debug(proc.stdout)
-    if field_set.output_path:
+    if field_set.output_path and field_set.output_path.value is not None:
```

Comment:
is this... an issue? I thought that the field shouldn't be Truthy if the value is None?